### PR TITLE
Add April Monthly Update

### PR DESCRIPTION
**Pull request by lauragift21**
Draft for RubyGems Monthly Update.

Hi @deivid-rodriguez can you help review the RubyGems News section?

### CODE REVIEW

File: _posts/2022-05-08-april-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+# RubyGems News
+
+In April, we released new versions of RubyGems [3.3.11](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3311--2022-04-07), [3.3.12](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3312--2022-04-20) and Bundler [2.3.11](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2311-april-7-2022), [2.3.12](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2312-april-20-2022).
+
+The following is a non-exhaustive list of the improvements included in the above releases (see the changelog for further information):
+
+- added modern rubies as valid platform values in Gemfile DSL - [#5469](https://github.com/rubygems/rubygems/pull/5469)
+- fixed missing RubyGems version when using old APIs - [#5496](https://github.com/rubygems/rubygems/pull/5496).
```
Comment (deivid-rodriguez) [inline _posts/2022-05-08-april-rubygems-updates.md:18]: This one landed by the end of March, but was not part of any of the above releases, so I think it's best to include it in the following update.

File: _posts/2022-05-08-april-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in April.
+
+
+# RubyGems News
+
+In April, we released new versions of RubyGems [3.3.11](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3311--2022-04-07), [3.3.12](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3312--2022-04-20) and Bundler [2.3.11](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2311-april-7-2022), [2.3.12](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2312-april-20-2022).
+
```
Comment (deivid-rodriguez) [inline _posts/2022-05-08-april-rubygems-updates.md:14]: I would add a separate paragraph on top of this to highlight the addition of experimental support for rust extensions, and remove that entry from the list above.

Something like this, for example:

> As the main highlight of this month, RubyGems now has experimental builtin support for Rust extensions, thanks to the work at https://github.com/rubygems/rubygems/pull/5175.

File: _posts/2022-05-08-april-rubygems-updates.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+The following is a non-exhaustive list of the improvements included in the above releases (see the changelog for further information):
```
Comment (deivid-rodriguez) [inline _posts/2022-05-08-april-rubygems-updates.md:15]: If we add the above paragraph, we can tweak this like

```suggestion
The following is a non-exhaustive list of other improvements included in the above releases (see the changelog for further information):
```

**Review by deivid-rodriguez — COMMENTED** [review]
I added a few comments, looks good!

File: _posts/2022-05-08-april-rubygems-updates.md (hunk 2 above)
Comment (lauragift21) [reply _posts/2022-05-08-april-rubygems-updates.md:14]: Got it! Thank you.

**Review by deivid-rodriguez — APPROVED** [review]

**Pull request merged**